modules/graphql/server-ts/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@gqlapp/graphql-types-server-ts",
+  "name": "@gqlapp/graphql-server-ts",
```

Comment:
Should we really merge `graphql-types` and `graphql` modules together? 

The other consideration is the name of the module `graphql` it is too ambigious if we just want to move GraphQL server-side setup here. Maybe we should name module `graphql-server` then?

Reply:
@vlasenko personally I think that it is quite reasonable. I don't think we will need graphql types without using gpaphql in general. Also, it allows keeping things in one place which is more comfortable.

Reply:
@evgenijolshanskij I don't have strong opinion here, so I'm okay with this merge. How about `graphql` name? Right now we have server setup there, so it should be named `graphql-server` maybe?

Reply:
The name would look like "@gqlapp/graphql-server-server-ts" because of the name convention if we rename the folder.

Reply:
@basilalex yes

Reply:
Done.